bin/assets/styles/report.css

```
@@ -0,0 +1,10 @@
+#report {
+	position: absolute;
+	top: 0.5em;
+	right: 0.5em;
+	width: 1em;
```

```suggestion
	width: 1.5em;
```
c'était un peu trop petit